hive_agent/store/store.py

```diff
@@ -1,36 +1,41 @@
+import logging
+import sys
 from datetime import datetime
 
 from sqlalchemy import delete, update
 from sqlalchemy.future import select
 
 from hive_agent.store import DataEntry
 
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
```

Comment:
keep this as INFO because we don't want to see the debug logs yet. we'll set the log level as environment variables.